src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -3069,10 +3069,9 @@ public void WritingHugeBase64Bytes(JsonWriterOptions options)
         }
 
         // https://github.com/dotnet/runtime/issues/30746
-        [Theory]
+        [Theory, OuterLoop("Very long running test")]
```

Comment:
This test to took 40 seconds to run on my machine, arguably this is too long even for outerloop. I think we could just reduce the amount of inputs being used by this theory.

Reply:
Thanks for removing the conditions for the outer loop attribute. It's also showing up in the output I shared for the issue this is fixing.